log/testutil.go

```diff
@@ -34,15 +34,42 @@ func interestingGoroutines() (gs []string) {
 			strings.Contains(stack, "stackimpact-go/internal") ||
 			// stackimpact uses persistent http client conns
 			strings.Contains(stack, "created by net/http.(*Transport).dialConn") ||
-			strings.Contains(stack, "log.MainTest") {
+			strings.Contains(stack, "log.MainTest") ||
+			matchesUserInterestingGoroutine(stack) {
 			continue
 		}
+
 		gs = append(gs, stack)
 	}
 	sort.Strings(gs)
 	return
 }
 
+var userInterestingGoroutines []string
```

Comment:
They are actually UNinteresting goroutines...

Reply:
Yup, I inadvertently inverted the meaning...